### PR TITLE
Umccr demo admin

### PR DESCRIPTION
**Pull request by andrewpatto**

### CODE REVIEW

**Comment by andrewpatto** [comment]
Policy plan is as needed

```
+ policy      = jsonencode(
            {
              + Statement = [
                  + {
                      + Action   = "s3:GetBucketLocation"
                      + Effect   = "Allow"
                      + Resource = [
                          + "arn:aws:s3:::umccr-10g-data-dev",
                          + "arn:aws:s3:::umccr-10f-data-dev",
                        ]
                    },
                  + {
                      + Action   = [
                          + "s3:GetObjectTagging",
                          + "s3:GetObject",
                        ]
                      + Effect   = "Allow"
                      + Resource = [
                          + "arn:aws:s3:::umccr-10g-data-dev/*",
                          + "arn:aws:s3:::umccr-10f-data-dev/CHINESE/*",
                          + "arn:aws:s3:::umccr-10f-data-dev/ASHKENAZIM/*",
                        ]
                    },
                ]
              + Version   = "2012-10-17"
            }
        )
```

File: terraform/stacks/umccr_demo_admin/main.tf [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+locals {
+  common_tags = {
+    "Environment" : "demo",
+    "Stack" : "umccr_demo_admin",
```
Comment (reisingerf) [inline terraform/stacks/umccr_demo_admin/main.tf:30]: Not sure we settled on it yet, but I think we should adopt the same `org.umccr:` prefix for those two.

File: terraform/stacks/umccr_demo_admin/main.tf [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    "org.umccr:Product" : "ElsaData"
```
Comment (reisingerf) [inline terraform/stacks/umccr_demo_admin/main.tf:31]: The idea was that we accumulate all resources that need `PlatformOwner` permissions in this TF stack. As such I'd keep the Elsa specific tag(s) outside the `common_tags`.

Comment (andrewpatto) [reply terraform/stacks/umccr_demo_admin/main.tf:31]: ah ok - sure. I thought it was just elsa stuff

Comment (andrewpatto) [reply terraform/stacks/umccr_demo_admin/main.tf:31]: Should I introduce tags for Elsa specifically on the user and related objects?

**Comment by victorskl** [comment]
ditto context convo
- https://umccr.slack.com/archives/C02LGHE2G/p1694054361020339
- https://umccr.slack.com/archives/C09FPTN4E/p1694137684298209

**Comment by victorskl** [comment]
[UoM AWS Migration: Create new AWS account for dedicated Demo stack](https://trello.com/c/2peVjtHU/1537-uom-aws-migration-create-new-aws-account-for-dedicated-demo-stack)

**Comment by victorskl** [comment]
Just wanna say that, I will follow up on tag prefix `org.umccr:` on my side as well.

**Review by victorskl — APPROVED** [review]
LGTM

**Review by reisingerf — APPROVED** [review]

**Comment by reisingerf** [comment]
@andrewpatto if you merge this I'm happy to deploy it to the demo account. No urgency from my side though.

**Comment by andrewpatto** [comment]
> @andrewpatto if you merge this I'm happy to deploy it to the demo account. No urgency from my side though.

Yes please - we just got the approval for MCRI/AG logins - so I will deploy the demo now - and make the object signing work once this comes through.

**Pull request merged**